CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### ✨ New features
 
 - *...Add new stuff here...*
+- [android] Do not crash when OEMs return 4 sensor values instead of 3 for accelerometer sensor changes.
```

Comment:
I meant this changelog https://github.com/HudHud-Maps/maplibre-native/blob/main/platform/android/CHANGELOG.md I should have been more specific, the release workflow checks if the new version has a changelog entry or not. I should change it at some point so that we don't have to bother with that.